packages/block-editor/README.md

```diff
@@ -517,6 +517,18 @@ _Returns_
 
 -   `string`: returns the cssUnit value in a simple px format.
 
+### getSpacingPresetCssVar
+
+Converts a spacing preset into a custom value.
```

Comment:
Shouldn't this be something like "Converts a spacing preset into a preset CSS custom property"? I was reading "custom value" as an actual custom value, like you'd set with the unit control.